core/src/main/scala/org/apache/spark/storage/PushBasedFetchHelper.scala

```diff
@@ -138,10 +140,19 @@ private class PushBasedFetchHelper(
         ((shuffleBlockId.shuffleId, shuffleBlockId.reduceId), size)
     }.toMap
     val address = req.address
+    var remainingBlockCount = sizeMap.size
```

Comment:
This I think is not needed. This is different from sending `FetchShuffleBlock` requests where multiple blockIds are included in a network request. For merged meta request, each network request is for just one meta request.

Reply:
As per https://github.com/apache/spark/pull/33446#discussion_r674418174 this has been reverted, so resolving the comment